recipe/build.sh

```diff
@@ -58,6 +59,11 @@ sed -e "s|\${PREFIX}|${PREFIX}|g" -e "s|\${x11_inc}|${x11_inc}|g" -e "s|\${x11_l
 echo -e "n\n" | ./Configure
 make Everything
 
+if [ ! -f "${PREFIX}/bin/ncl" ]; then
```

Comment:
I wonder if running `${PREFIX}/bin/ncl -V` would be a more thorough check to see if the binary even runs instead of it just existing?  I agree that what you currently have is very important already!

Reply:
It would, but that would not work for cross-compilation. Indeed, that is usually the reason why tests are not run.

Reply:
Oh my, thanks for reminding me of that!